app/views/links/property.erb

```diff
@@ -0,0 +1,9 @@
+<form action='/property/new' method='get'>
```

Comment:
Please can we add a line here linking to a separate erb file which welcomes the user, as per the sketches. We'll discuss.